js/src/grpc-client.ts

```diff
@@ -337,10 +328,9 @@ export class DateiLagerGrpcClient {
 
   /**
    * Update an object.
-   *
    * @param project The id of the project.
    * @param obj     The object to update.
-   * @returns         The latest project version or `null` if something went wrong.
+   * @returns       The latest project version or `null` if something went wrong.
```

Comment:
I have no idea why but my and Scott's auto formatter was setting it up this way.